API_Kraken/get_bitcoin_price.py

```diff
@@ -1,20 +1,35 @@
 import requests
 import datetime
 import click
+import time
 
 @click.command()
 @click.option('--data_file', help = 'Path to datafile.')
-def get_price(data_file):
+@click.option('--break_time', default = 5, help = 'Time of a break between prices.')
+
+def get_price():
     try:
         get_request = requests.get('https://api.kraken.com/0/public/Ticker?pair=XBTUSD')
         price = get_request.json()['result']['XXBTZUSD']['a'][0]
         date = datetime.datetime.now().strftime('%c')
-        value = (price + ' USD, ' + date)
-        print(value)
     except Exception as e:
         raise Exception(f'An error has occured due to: {e}')
+    return price, date
+
+def append_data_file(data_file, get_price):
+    with open(data_file, 'a') as file:
+        file.write('\n' + get_price)
+
+def remove_data_file(data_file):
     with open(data_file, 'a') as file:
-        file.write('\n' + repr(price) + ' USD, ' + repr(date))
+        file.seek(0)
+        file.truncate()
+        file.write("Bitcoin prices for: " + datetime.datetime.now().strftime('%x'))
 
-if __name__ == '__main__':
-    get_price()
\ No newline at end of file
+def main(break_time, data_file):
+    remove_data_file(data_file)
+    while True:
+        print(get_price())
+        append_data_file(data_file)
```

Comment:
append_data_file accepts datafile and get_price arguments. Get_price returns two values price and date.

This code should look like:

```
   price, date = get_price()
        append_data_file(data_file,price)
```